docs/source/deploying-volttron/multi-platform/multi-platform-rabbitmq-deployment.rst

```diff
@@ -212,15 +204,12 @@ upstream servers on the downstream server and make the VOLTTRON exchange
 
    a. On the downstream server run a listener agent which subscribes to messages from all platforms
 
-     - Open the file: examples/ListenerAgent/listener/agent.py. Search for ``@PubSub.subscribe('pubsub', '')`` and replace that line with ``@PubSub.subscribe('pubsub', 'devices', all_platforms=True)``
-     - upgrade the listener
-
-         .. code-block:: bash
+       .. code-block:: bash
 
-            scripts/core/upgrade-listener
+          scripts/core/upgrade-listener
 
 
-   b. Install platform driver, configure fake device on upstream server and start volttron and platform driver. ``vcfg --agent platform_drive
+   b. Install platform driver, configure fake device on upstream server and start volttron and platform driver.
```

Comment:
Do we really need to test with master driver and fake device for everything? It is a pain to do it since I have to stop volttron, run vcfg and restart volttron. All our documents refer to installing master driver and none states that we are simply using that as a producer of messages to forward. So all documents reads as if master driver install is required. 
 If two instances are federated, I should be able to forward any topic, right? Now that all agents produce heartbeat, could we just forward heartbeat message for test

Reply:
Heartbeat messages are difficult to spot. Master-driver agent is easy to setup and forwarding is easy to spot. I would rather explain with master driver agent.